### ISSUE TITLE
npm reports severity vulnerabilities in `glob-stream` version 

### ISSUE THREAD

**Issue by Drogglbecher** [microsoft/parallel-prettier#14]
When adding the latest version of `parallel-prettier` in the `package-json` of my project, npm tells me the following while doing `npm audit`:

```
# npm audit report

glob-parent  <5.1.2
Severity: moderate
Regular expression denial of service - https://npmjs.com/advisories/1751
No fix available
node_modules/glob-stream/node_modules/glob-parent
  glob-stream  >=5.3.0
  Depends on vulnerable versions of glob-parent
  node_modules/glob-stream
    @mixer/parallel-prettier  *
    Depends on vulnerable versions of glob-stream
    node_modules/@mixer/parallel-prettier

3 moderate severity vulnerabilities

Some issues need review, and may require choosing
a different dependency.
```

**Comment by trivikr** [comment]
Looks like the dependency on `glob-stream` needs to be bumped which gets vulnerable `glob-parent`.

The latest version is [glob-stream@7](https://www.npmjs.com/package/glob-stream) as of June 2022, and parallel-prettier depends on the latest minor version of v6 https://github.com/microsoft/parallel-prettier/blob/79515f9f7daf568133cb05078c38cc20c4789921/package.json#L37

The dependency `glob-stream@7` bumped their dependency on `glob-parent` from v3 to v6.
[gulpjs/glob-stream@`v6.1.0...v7.0.0`#diff-7ae45ad102](https://github.com/gulpjs/glob-stream/compare/v6.1.0...v7.0.0#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519)

**Comment by trivikr** [comment]
Thanks for merging the PR.

Is `v2.0.3` or `v2.1.0` release planned to publish it?

**Comment by trivikr** [comment]
> Is `v2.0.3` or `v2.1.0` release planned to publish it?

The fix was released in [`v2.0.3`](https://github.com/microsoft/parallel-prettier/releases/tag/v2.0.3)
